docs/intellij-idea-plugin-guide/master.adoc

```diff
@@ -1,13 +1,16 @@
-
 :toc:
 :toclevels: 4
 :numbered:
 
-include::topics/templates/document-attributes.adoc[]
+
 
 :imagesdir: topics/images
-:context: idea-plugin
-:idea-plugin:
+:context: idea-plugin-guide
```

Comment:
You need to update the context attribute so that it matches your modules: intellij-idea-plugin-guide